javascript/node/selenium-webdriver/chrome.js

```diff
@@ -727,6 +737,37 @@ class Driver extends webdriver.WebDriver {
         new command.Command(Command.LAUNCH_APP).setParameter('id', id),
         'Driver.launchApp()');
   }
+  
+  /**
+   * Schedules a command to get Chrome network emulation settings.
+   * @return {!promise.Thenable<T>} A promise that will be resolved
+   *     when network emulation settings are retrievied.
+   */
+  getNetworkConditions() {
+    return this.schedule(
+        new command.Command(Command.GET_NETWORK_CONDITIONS),
+        'Driver.getNetworkConditions()');
+  }
+
+  /**
+   * Schedules a command to set Chrome network emulation settings.
+   * @param {number} latency Additional latency (ms).
+   * @param {number} download_throughput Maximal aggregated download throughput. 
+   * @param {number} upload_throughput Maximal aggregated upload throughput.
+   * @return {!promise.Thenable<void>} A promise that will be resolved
+   *     when network emulation settings are set.
+   */
+  setNetworkConditions(latency, download_throughput, upload_throughput) {
```

Comment:
Yeah, since the parameters are 3 numbers I think it will be difficult for users to remember the order.

Also, it looks like chromedriver accepts `throughput` when `download_throughput` and `upload_throughput` are the same.